pkg/state/state_run.go

```diff
@@ -122,9 +122,9 @@ func SortedReleaseGroups(releases []Release, reverse bool) ([][]Release, error)
 	}
 
 	if reverse {
-		sort.Slice(groups, func(i, j int) bool {
-			return j < i
-		})
+		for i, j := 0, len(groups)-1; i < j; i, j = i+1, j-1 {
```

Comment:
If I may `i++, j--` might be a bit clearer than `i, j = i+1, j-1` no?

also... a test? :) [repro for #1775 ]

Reply:
I forgot there is no , there in golang (old C habits die hard) so it would need to be something like
```golang
		for i, j := 0, len(groups)-1; i < j; i++ {
			groups[i], groups[j] = groups[j], groups[i]
			j--
		}
```
which is clearer to me but asymmetrical, so ignore me :-)

ps: tested with #1775 and this branch works so I guess maybe no additional test needed and just a quick patch release 0.138.8 would be most welcome - thanks!